### PR TITLE
[Look&Feel] use standard paragraph size

### PR DESCRIPTION
**Pull request by zhongnansu**
### Description
- update some instances where it's using inline styling for fonts, to use <EuiText size="s">, so it will work with different themes
- update existing paragraph/freeform text to have denser look by using `EuiText size="s".  For those which are already using `size="xs"`, remain unchanged.

image

 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by zhongnansu** [comment]
@lezzago @bowenlan-amzn Hi Ashish and Bowen, could any of you help with re-run the failed CI? Thanks

**Comment by bowenlan-amzn** [comment]
@amsiglan please review

File: public/pages/Dashboard/containers/__snapshots__/Dashboard.test.js.snap
Comment (virajsanghvi) [inline public/pages/Dashboard/containers/__snapshots__/Dashboard.test.js.snap:1498]: not you: but weird that this is basically the same style twice - could simply by removing a level if there's no risk, but up to you

File: public/pages/Dashboard/containers/__snapshots__/Dashboard.test.js.snap
Comment (virajsanghvi) [inline public/pages/Dashboard/containers/__snapshots__/Dashboard.test.js.snap:3456]: same as above and another instance below

**Review by virajsanghvi — APPROVED** [review]

File: public/pages/Dashboard/containers/__snapshots__/Dashboard.test.js.snap
Comment (zhongnansu) [reply public/pages/Dashboard/containers/__snapshots__/Dashboard.test.js.snap:1498]: So one of the `EuiText` is from `EuiEmptyPromt`, and it comes default as size="m", this is not exposed as an adjustable prop of `EuiEmptyPromt`. Only title size is exposed. https://oui.opensearch.org/1.5/#/display/empty-prompt

So if we still want to have the small text without updating Oui, I think this might be the only way to go.

Comment (virajsanghvi) [reply public/pages/Dashboard/containers/__snapshots__/Dashboard.test.js.snap:1498]: Sounds good - also missed this was a snapshot file

**Review by virajsanghvi — APPROVED** [review]
Approving assuming cypress test is not failing due to your changes

**Comment by zhongnansu** [comment]
@amsiglan @bowenlan-amzn @riysaxen-amzn Hi maintainers, PR has been approved by @virajsanghvi , could you help with approval , merge, and backport? Also the failed CI is not related to the change in this PR. Thanks.

**Review by amsiglan — APPROVED** [review]

**Review by riysaxen-amzn — APPROVED** [review]

**Review by AWSHurneyt — APPROVED** [review]
The cypress test changes should be unrelated to these changes. Approving.

**Pull request merged**
